src/chain/chain.rs

```diff
@@ -66,6 +66,8 @@ pub struct Chain {
     churn_in_progress: bool,
     /// Marker indicating we are processing a relocation.
     relocation_in_progress: bool,
+    /// Marker indicating that elders may need to change,
```

Comment:
can the comment be more explicit, say `Marker indicating elders may need to change, as there is membership change`

Reply:
It may be because of anything, that would change elders, in particular, if a member become Relocating. so I'm not sure it make sense to be more specific? (It just happen that both membership change and age increase happen on the same events)